features/support/env.rb

```diff
@@ -3,7 +3,7 @@
 
 Before do
   puts "Killing server..."
-  system "for i in `ps axu | grep 'server.rb' | awk '{print $2}'`; do kill -9 $i; done > /dev/null 2>&1"
+  system "for i in `ps axu | grep 'server.rb' |  awk '{print $2}'`; do kill -9 $i; done > /dev/null 2>&1"
```

Comment:
Why extra space?

Reply:
Unwanted, good catch. Probably added by accident. I might fix it in later PR instead of this one as it shouldn't cause anything.